Defs/ThingDefs_Misc/Weapons_Turrets.xml

```diff
@@ -237,7 +237,7 @@
         <warmupTime>1.3</warmupTime>
         <range>75</range>
         <ticksBetweenBurstShots>6</ticksBetweenBurstShots>
-        <burstShotCount>10</burstShotCount>
```

Comment:
I'm quite opposed to this, the 5 round burst is enough for ammo conservation purposes and the 10 round burst is handy for "oh shit!" moments

Reply:
Fine, reversed